### PR TITLE
[Playwright] Fix env export issue

### PR DESCRIPTION
**Pull request by aweng98**
Node fetch error due to missing RGP Auth0 credential because `export-playwright-env.sh` fails to evaluate `<< parameters.env >>`.

RGP test fails due ot missing Auth0 credentials.

```
adult-enrollment.spec.ts:22:3 Can complete application @functional @enrollment @rgp
[chromium] › tests/rgp/adult-enrollment.spec.ts:22:3 › Adult Self Enrollment › Can complete application @functional @enrollment @rgp 

    TypeError: fetch failed

        at Object.fetch (node:internal/deps/undici/undici:14062:11)
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at setAuth0UserEmailVerified (/home/circleci/repo/playwright-e2e/utils/api-utils.ts:134:49)
        at /home/circleci/repo/playwright-e2e/tests/rgp/adult-enrollment.spec.ts:57:5

ERROR: POST /oauth/token:
 TypeError: fetch failed
[90m    at Object.fetch (node:internal/deps/undici/undici:14062:11)[39m
[90m    at processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
    at setAuth0UserEmailVerified [90m(/home/circleci/repo/playwright-e2e/[39mutils/api-utils.ts:134:49[90m)[39m
    at [90m/home/circleci/repo/playwright-e2e/[39mtests/rgp/adult-enrollment.spec.ts:57:5
    at [90m/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/workerRunner.js:376:9
    at TestInfoImpl._runFn [90m(/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/testInfo.js:146:7[90m)[39m
    at [90m/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/workerRunner.js:331:26
    at TimeoutRunner.run [90m(/home/circleci/repo/playwright-e2e/[39mnode_modules/[4mplaywright-core[24m/lib/utils/timeoutRunner.js:45:14[90m)[39m
    at TimeoutManager.runWithTimeout [90m(/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/timeoutManager.js:62:7[90m)[39m
    at TestInfoImpl._runWithTimeout [90m(/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/testInfo.js:135:26[90m)[39m
    at WorkerRunner._runTest [90m(/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/workerRunner.js:312:5[90m)[39m
    at WorkerRunner.runTestGroup [90m(/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/workerRunner.js:192:11[90m)[39m
    at process.<anonymous> [90m(/home/circleci/repo/playwright-e2e/[39mnode_modules/[4m@playwright[24m/test/lib/worker.js:74:5[90m)[39m {
  cause: Error: getaddrinfo ENOTFOUND oauth
  [90m    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)[39m {
    errno: [33m-3008[39m,
    code: [32m'ENOTFOUND'[39m,
    syscall: [32m'getaddrinfo'[39m,
    hostname: [32m'oauth'[39m
  }
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -598,6 +598,8 @@ jobs:
           name: Save Vault secrets into temp file to persist across jobs
           command: |
             set -u
+            echo "export ENV=<< parameters.env >>" >> $BASH_ENV
+            source $BASH_ENV
```
Comment (aweng98) [inline .circleci/config.yml:602]: Create a new env variable `ENV` equals to `<< parameters.env >>` so `export-playwright-env.sh` can use the env variable.

File: .circleci/export-playwright-env.sh [hunk 2]
```diff
@@ -3,68 +3,68 @@
 mkdir -p playwright-env
 
 export sitePwd=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.sitePassword")
-echo "export SITE_PASSWORD=$sitePwd" >>playwright-env/envvars
+echo "export SITE_PASSWORD=$sitePwd" >> playwright-env/envvars
```
Comment (aweng98) [inline .circleci/export-playwright-env.sh:6]: Insert whitespace after `>>`.

File: .circleci/export-playwright-env.sh [hunk 3]
```diff
@@ -3,68 +3,68 @@
...
 # Read Auth0 RGP client credentials
-export rgpDomain=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .domain")
-export rgpAudience=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .audience")
-export rgpClientId=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .clientId")
-export rgpClientSecret=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"<< parameters.env >>\") | .clientSecret")
-echo "export RGP_AUTH0_DOMAIN=$rgpDomain" >>playwright-env/envvars
-echo "export RGP_AUTH0_AUDIENCE=$rgpAudience" >>playwright-env/envvars
-echo "export RGP_AUTH0_CLIENT_ID=$rgpClientId" >>playwright-env/envvars
-echo "export RGP_AUTH0_CLIENT_SECRET=$rgpClientSecret" >>playwright-env/envvars
+export rgpDomain=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.auth0 | .[] | select(.app==\"rgp\" and .env==\"$ENV\") | .domain")
```
Comment (aweng98) [inline .circleci/export-playwright-env.sh:27]: Replace `<< parameters.env >>` with `$ENV`.

Comment (Rinchan20) [reply .circleci/export-playwright-env.sh:27]: I think that makes it more read-able - looks nice

**Review by Rinchan20 — APPROVED** [review]

**Pull request merged**